### PR TITLE
Update eventstore to new version and url formats

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/eventstore.rb [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
+  url "https://eventstore.org/downloads/macos/EventStore-OSS-MacOS-macOS-v#{version}.tar.gz"
   name 'Event Store'
   homepage 'https://eventstore.org/'
 
-  binary "EventStore-OSS-MacOSX-v#{version}/eventstore"
-  binary "EventStore-OSS-MacOSX-v#{version}/eventstore-testclient"
+  binary "EventStore-OSS-MacOS-macOS-v#{version}/eventstore"
+  binary "EventStore-OSS-MacOS-macOS-v#{version}/eventstore-testclient"
 
   # shim script (https://github.com/Homebrew/homebrew-cask/issues/18809)
```
Comment (vitorgalvao) [inline Casks/eventstore.rb]: Link was correct.

File: Casks/eventstore.rb [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
-  eventstore_shimscript = "#{staged_path}/EventStore-OSS-MacOSX-v#{version}/eventstore"
-  testclient_shimscript = "#{staged_path}/EventStore-OSS-MacOSX-v#{version}/eventstore-testclient"
+  eventstore_shimscript = "#{staged_path}/EventStore-OSS-MacOS-macOS-v#{version}/eventstore"
+  testclient_shimscript = "#{staged_path}/EventStore-OSS-MacOS-macOS-v#{version}/eventstore-testclient"
 
   preflight do
     IO.write eventstore_shimscript, <<~EOS
```
Comment (vitorgalvao) [inline Casks/eventstore.rb]: Was also correct. See https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/eventstore.rb (hunk 1 above)
Comment (ChrisChinchilla) [reply Casks/eventstore.rb]: @vitorgalvao But isn't the latest version, the changes made are the latest version.

File: Casks/eventstore.rb (hunk 2 above)
Comment (ChrisChinchilla) [reply Casks/eventstore.rb]: @vitorgalvao But isn't the latest version, the changes made are the latest version. And I'm not sure if I understand what the link referenced has to do with the update tarball link?

File: Casks/eventstore.rb (hunk 1 above)
Comment (ChrisChinchilla) [reply Casks/eventstore.rb]: Ah, you're referring to line 12, sorry GH diff viewer showing something else.

File: Casks/eventstore.rb (hunk 2 above)
Comment (ChrisChinchilla) [reply Casks/eventstore.rb]: Sorry, diff viewer strikes again

**Comment by vitorgalvao** [comment]
Thank you and welcome.

**Pull request merged**